README.md

```diff
@@ -1,2 +1,22 @@
 # zap-cli
-A Node.js CLI command for searching for, or opening a YouTube video with Zen Audio Player.
+A Node.js CLI tool for searching for, or opening a YouTube video with [Zen Audio Player](https://github.com/zen-audio-player/zen-audio-player.github.io). To use the zap-cli command-line tool, clone the repository and run a global npm install inside of it.
+
+```
+git clone https://github.com/zen-audio-player/zap-cli && cd zap-cli
+npm install -g
+```
+
+You can also access the zap command-line tool via the [zap-cli npm package](
```

Comment:
Can you move the npm related content to another branch? (And subsequently, a new pr) I want to merge everything in that's 100% true right now, then before we publish to npm I can merge in the changes relating to npm installation/usage